README.md

```diff
@@ -22,14 +20,12 @@ No Outputs
 
 ### Variables File
 
-```json
-{
-    "Environment": "Dev",
-    "Version": "1.3.62",
-    "LaunchDarklyKey": "abc",
-    "GoogleAnalyticsKey": "123",
-    "AppInsightsKey": "a1b2c3",
-}
+```yaml
+Environment: Dev
+Version: 1.3.62
+LaunchDarklyKey: abc
```

Comment:
Do you want to keep LaunchDarklyKey here since you're providing it as an environment variable in the example?

Reply:
Yeah, I wanted to show that the environment variable would override the file. I don't know if I actually mentioned that behavior in the readme though...I'll have to add it if it isn't there already.